interfaces/builtin/steam_support.go

```diff
@@ -395,6 +253,27 @@ type steamSupportInterface struct {
 	commonInterface
 }
 
+func (iface *steamSupportInterface) AppArmorConnectedPlug(spec *apparmor.Specification, plug *interfaces.ConnectedPlug, slot *interfaces.ConnectedSlot) error {
+	// if apparmor supports "allow all" then use it. This allows not updating
+	// the supported features list as new features are added.
+	if apparmor_sandbox.ProbedLevel() != apparmor_sandbox.Unsupported {
+		features, err := apparmor_sandbox.ParserFeatures()
+		if err != nil {
+			return err
+		}
```

Comment:
Do we need mqueue here? Without the min version feature, what will happen in the worse case if we have it here?